samples/meetings-visual-indicator/nodejs/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "visual-indicator-in-meetings",
+  "version": "1.0.0",
+  "description": "Visual Indicator in meetings notification",
```

Comment:
Visual Indicator in Teams meetings notification

Reply:
fixed as per comment